### DISCUSSION TITLE
3.0.5使用ui包后，对主题的自定义配置似乎无效

### DISCUSSION THREAD

**Discussion by mas-alone** [libsgh/PanIndex#268] [Q&A]
之前2.0.9是生效的，不知道是不是我改错了。
./templates/pan/mdui/index.html

使用
								{{- if $.has_parent}}
								<a id="share" href="javascript:void(0)" mdui-menu="{target: '#share-menu',covered: false}" class="mdui-btn mdui-btn-icon" mdui-tooltip="{content: '分享'}"><i class="mdui-icon material-icons">share</i></a>
								<ul id="share-menu" class="mdui-menu" data-fp="{{ $.full_path }}" data-file-type="0" data-aid="{{ $.account.Id }}">
									<li class="mdui-divider"></li>
									<li class="mdui-menu-item">
										<img id="qrcode" src="" width="100%" />
									</li>
									<li class="mdui-divider"></li>
									<li class="mdui-menu-item mdui-text-center">
										<button id="copyShortUrl" class="mdui-btn mdui-btn-raised mdui-ripple mdui-btn-dense mdui-color-indigo">复制短链接</button>
									</li>
								</ul>
								{{- end}}
								`

**Comment by libsgh** [comment]
v3.0.5的bug，更新到v3.0.6解决

**Reply by mas-alone** [reply]
已解决！辛苦开发者了，十分优秀的项目，加油！
